packages/templates/package.json

```diff
@@ -46,7 +46,7 @@
     "@googleforcreators/date": "*",
     "@googleforcreators/elements": "*",
     "@googleforcreators/stickers": "*",
-    "puppeteer": "*"
```

Comment:
This can actually stay `*`. The version in the main `package.json` is what matters.